web-app/packages/admin-app/src/router.ts

```diff
@@ -144,6 +145,16 @@ export const createRouter = (pinia: Pinia) => {
           header: AppHeader
         },
         props: true
+      },
+      {
+        path: '/overview',
```

Comment:
could you add this also to enterprise implementation and try If everything is fine ?